cmd/manager/main.go

```diff
@@ -78,15 +77,15 @@ func main() {
 	// Get a config to talk to the apiserver
 	cfg, err := config.GetConfig()
 	if err != nil {
-		log.Error(err, "")
+		log.Error(err, "Failed to get the config for talking to a Kubernetes API server")
```

Comment:
Should we start with `capitals`? The example I see here seems to say [otherwise](https://github.com/go-logr/logr/blob/a1ebd699b1950beb1da0752cbb5559662018f798/examples/usage_example.go#L121)

Reply:
done, changed log.INFO to smaller case as well following the example.